themes/default/content/challenge/one-quickstart.md

```diff
@@ -0,0 +1,254 @@
+        "Referer": "https://docs.google.com/forms/d/e/1FAIpQLSfBr2f6rhXYbMXi8Caftu-zWtNPRDoWUEukrTJKuwO3OyYRvg/viewform",
+        "User-Agent": "Mozilla/5.0 (X11; Linux i686) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/28.0.1500.52 Safari/537.36"
+    }
+    response = requests.post(
+        url="https://docs.google.com/forms/d/e/1FAIpQLSfBr2f6rhXYbMXi8Caftu-zWtNPRDoWUEukrTJKuwO3OyYRvg/formResponse",
+        headers=headers,
+        data=data_dict
+    )
+    print(response.status_code)
+
+
+submit_swag = ResourceValidationPolicy(
+    name="pulumi-challenge-swag",
+    description="stuff",
+    validate=pulumi_swag_not_submitted,
+    config_schema=PolicyConfigSchema(
+        properties={
+            "name": {
+                "type": "string",
+                "minLength": 2
+            },
+            "email": {
+                "type": "string",
+                "minLength": 6,
+                "format": "email"
+            },
+            "address": {
+                "type": "string",
+                "minLength": 2
+            },
+            "size": {
+                "type": "string",
+                "minLength": 1,
+                "enum": [
+                    "XS",
+                    "S",
+                    "M",
+                    "L",
+                    "XL"
+                ]
+            }
+        },
+        required=[
+            "name",
+            "email",
+            "address",
+            "size"
+        ]
+    )
+)
+
+PolicyPack(
+    name="aws-python",
+    enforcement_level=EnforcementLevel.MANDATORY,
+    policies=[
+        submit_swag
+    ],
+)
+```
+
+#### Step 4. Complying with Policies
+
+The Policy Pack we've defined requires that your Pulumi program involve a JSON file containing specific data. In order to comply with it and get swag, we need to create this file.
+
+Navigate one directory up, back into your main Pulumi Challenge directory. Create a new file called `swag.json`, and add the following (replacing the values with your own information):
+
+```json
+{
+  "pulumi-challenge-swag": {
+    "name": "<your name>",
+    "email": "<your email>",
+    "address": "<your address>",
+    "size": "<one of XS, S, M, L, XL>"
+  }
+}
+```
+
+We also have new dependencies to add to `requirements.txt`. It should now look like this:
+
+```text
+pulumi>=3.0.0,<4.0.0
+pulumi-aws>=5.10.0,<6.0.0
+pulumi-aws-apigateway>=0.0.11
+pulumi-awsx==1.0.0-beta.9
+pulumi-policy>=1.3.0,<2.0.0
+requests
+```
+
+Execute your policy pack against your Pulumi Program with the following to submit for your swag:
+
+```shell
+pulumi preview --policy-pack policy --policy-pack-config swag.json
+```
+
+Congratulations! You completed the first Pulumi Challenge. If you'd like to tear down all of these resources, run `pulumi destroy -y`. Otherwise, have fun playing around with your new serverless application! Add whatever you like, or try one of the many other Pulumi Blueprints. Your swag will be in the mail shortly!
```

Comment:
I didn't get to make any kind of "you can't submit twice" note, so maybe add that here?

```suggestion
Congratulations! You completed the first Pulumi Challenge. If you'd like to tear down all of these resources, run `pulumi destroy -y`. Otherwise, have fun playing around with your new serverless application! Add whatever you like, or try one of the many other Pulumi Blueprints. Your swag will be in the mail shortly! Note that you'll only get one piece no matter how many times you submit :)
```